src/styles/transitions.spec.js

```diff
@@ -2,9 +2,25 @@
 /* eslint-env mocha */
 
 import { assert } from 'chai';
+import { spy } from 'sinon';
 import transitions, { easing, duration } from './transitions';
 
 describe('transitions', () => {
+  let originalConsoleError;
+
+  beforeEach(() => {
+    /* eslint-disable no-console */
+    originalConsoleError = console.error;
```

Comment:
I believe that flow won't complain when using the stub API. Having those tests is definitly valuable. The React codebase have a lot of them.